src/app/campaign/flight/flight-form.component.html

```diff
@@ -7,19 +7,43 @@
     <div class="inline-fields">
       <mat-form-field appearance="outline">
         <mat-label>Start Date</mat-label>
```

Comment:
My fault here screwing up these labels (I think I going back and further between the material datepickers and the styleguide prx-datepickers with the UTC support and everything,) but we should probably make the Actual Start and End date labels consistent.